sw/device/sca/ecc256_keygen_serial.c

```diff
@@ -60,6 +60,13 @@ enum {
    * Max number of traces per batch.
    */
   kNumBatchOpsMax = 256,
+  /**
+   * Number of cycles that Ibex should sleep to minimize noise during OTBN
+   * operations. Caution: This number should be chosen to provide enough time
+   * to complete the operation. Otherwise, Ibex might wake up while OTBN is
+   * still busy and disturb the capture.
+   */
+  kIbexOtbnSleepCycles = 800,
```

Comment:
Is this an estimated cycle count, or did you measure how many cycles the OTBN takes?

Reply:
For now I've just selected `800` as a placeholder value. It is enough to mute ibex for the 3500 sample measurements that we are currently running.
Eventually, we plan to make this value programmable by the `capture.py` script, but I wouldn't want to do that in this PR.